### ISSUE TITLE
Set up CBET Datacenter

### ISSUE THREAD

**Issue by ct-martin** [RITlug/config-management#14]
Meta-issue for setting up of CBET datacenter.

To-do:
* [ ] Install Proxmox on old servers
* [ ] Install Proxmox or oVirt on the newly-donated server
  * [ ] Network bridge for RIT-exposed network & another for everything behind firewall
  * [ ] Ceph for disks
* [ ] Get MAC registered (will require email to ITS contact)
* [ ] Set up pfSense for...
  * routing
  * DHCP
  * DNS
  * firewall
  * malicious IP/DNS blacklisting
  * certificate management
  * reverse-proxying
  * management VLAN
  * VPN
* [ ] Configure pfSense & switch to put the other servers behind pfSense (and by proxy the newly-donated server)
* [ ] Set up OpenShift for Docker-related hosting
  * oVirt storage backend for OpenShift/Kubernetes?
* [ ] Set up a FreeIPA server to reduce account management overhead (particularly for admin-ing)
* [ ] Test that things work
* [ ] Document _everything_ (see also: RITlug/runbook#47), including...
  * Configurations
    * Servers
      * Ansible playbooks, if applicable
    * Network
  * Root account passwords
  * Server specs
  * How to access
    * Digitally
    * Physically
    * CBET & ITS contact info location

**Comment by ct-martin** [comment]
I'm going to hide in the Lab for DA&A (formerly FET Lab) tomorrow and work on this.

**Comment by ct-martin** [comment]
Blocked by #17; network access was removed by the sysadmin and permission was given to power them down.

**Comment by jrtechs** [comment]
We should look at this again. Our two main working servers are in the ITS data center. In CBET there is a KVM, and some old non operational servers. We might want to look at re-shelving a working server there or just move out completely.

**Comment by Tjzabel** [comment]
CBET datacenter is a no-go in my opinion. I think we should pay a visit to see what we have, otherwise I am perfectly okay with either donating or getting rid of the servers if we don't think we have a viable use for them, given the fact that we have a completely available R900.

**Comment by ct-martin** [comment]
@Tjzabel if you are moving out of CBET you should email the admin and get remove what we still have there out

**Comment by Tjzabel** [comment]
@ct-martin yep, there is a new issue to follow that progress.
